src/main/java/org/apache/commons/imaging/palette/LongestAxisMedianCut.java

```diff
@@ -43,7 +43,7 @@ public boolean performNextMedianCut(final List<ColorGroup> colorGroups, final bo
                 && colorGroup.alphaDiff > colorGroup.redDiff
                 && colorGroup.alphaDiff > colorGroup.greenDiff
                 && colorGroup.alphaDiff > colorGroup.blueDiff) {
-            doCut(colorGroup, ColorComponent.ALPHA, colorGroups, ignoreAlpha);
+            doCut(colorGroup, ColorComponent.ALPHA, colorGroups, false);
```

Comment:
I think it was harmless to keep using `ignoreAlpha` here. There are other conditions in this if statement, and they are all using `ignoreAlpha`. Having that variable in all the conditions might make it easier to remove/update it later if necessary.

Reply:
HI @kinow 
Changed
TY